### PR TITLE
feat: new section order.

### PR DESCRIPTION
**Pull request by senyka33**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,50 +52,62 @@ <h2 class="advantage-item__title">Большой выбор</h2>
...
+        <div class="order-go">
+          <h2 class="order-go__title">Готовы сделать заказ?</h2>
+          <p class="order-go__text">Заполните простую форму для заказа и ожидайте SMS-подтверждение заказа на своём
+            телефоне!</p>
+          <a class="order-go__link" href="formazakaza.html">Форма для заказа</a>
         </div>
+        <div class="order-go__image">
+          <img class="order-go__image-cat" src="img/cat.png" alt="Лого котика">
+        </div>
+      </div>
```
Comment (emarf) [inline index.html]: don`t understand class naming

File: index.html [hunk 2]
```diff
@@ -52,50 +52,62 @@ <h2 class="advantage-item__title">Большой выбор</h2>
...
   </main>
-  <footer>
-    <div>
-      <p>Давайте дружить!</p>
-      <li><a href="https://www.instagram.com/kot_splushkin/">Instagram</a></li>
-      <li><a href="https://www.kufar.by/item/113966348">Куфар</a></li>
+  <footer class="footer">
+    <p class="footer-title">Давайте дружить!</p>
+    <div class="footer-item">
+      <a class="footer-button_link" href="https://www.instagram.com/kot_splushkin/">
```
Comment (emarf) [inline index.html]: just footer link

Comment (emarf) [reply index.html]: add target="_blank"

File: index.html [hunk 3]
```diff
@@ -52,50 +52,62 @@ <h2 class="advantage-item__title">Большой выбор</h2>
...
+        <img src="./img/instagram.png" class="footer-button_instagram">
+      </a>
+      <a class="footer-button_link" href="https://www.instagram.com/kot_splushkin/">
```
Comment (emarf) [inline index.html]: add target="_blank"

File: style.css [hunk 4]
```diff
@@ -43,6 +43,11 @@ body {
   padding: 10px 10px;
   border: none;
 }
+
+.common-button:hover {
+  background: #6b4913;
+  transition: all 0.2s ease-in-out;
```
Comment (emarf) [inline style.css]: move to .common-button and select what do you want animate(background, color, height, etc.)

File: style.css [hunk 5]
```diff
@@ -43,6 +43,11 @@ body {
   padding: 10px 10px;
   border: none;
 }
+
+.common-button:hover {
+  background: #6b4913;
```
Comment (emarf) [inline style.css]: better background-color

File: index.html [hunk 6]
```diff
@@ -52,50 +52,62 @@ <h2 class="advantage-item__title">Большой выбор</h2>
...
-          </div>
+      </div>
+    </section>
+    <section class="order">
+      <div class="order-item">
+        <div class="order-go">
+          <h2 class="order-go__title">Готовы сделать заказ?</h2>
+          <p class="order-go__text">Заполните простую форму для заказа и ожидайте SMS-подтверждение заказа на своём
+            телефоне!</p>
+          <a class="order-go__link" href="formazakaza.html">Форма для заказа</a>
```
Comment (emarf) [inline index.html]: you have .common-button
mb better to do smth like this
`<a>
   <button class="common-button">Text</button>
</a>`

File: style.css [hunk 7]
```diff
@@ -217,3 +227,98 @@ body {
...
+/* Section - READY ORDER */
+.order {
+  display: flex;
+  max-width: 960px;
+  height: 267px;
+  width: 100%;
+  flex-basis: 100%;
+  background-image: url("./img/background-order-go.png");
+  margin: 36px auto;
+  justify-content: space-between;
```
Comment (emarf) [inline style.css:254]: you need this?

File: style.css [hunk 8]
```diff
@@ -217,3 +227,98 @@ body {
...
+  border-radius: 5px;
+  box-shadow: 10px 10px 5px rgba(80, 79, 79, 0.5);
+}
+
+.order-item {
+  display: flex;
+  padding: 21px 68px 21px 325px;
```
Comment (emarf) [inline style.css]: i don`t like this solution

File: style.css [hunk 9]
```diff
@@ -217,3 +227,98 @@ body {
...
+}
+
+.order-go {
+  padding: 39px 71px 60px 0px;
```
Comment (emarf) [inline style.css]: i don`t like this solution

File: style.css [hunk 10]
```diff
@@ -217,3 +227,98 @@ body {
...
+  text-transform: uppercase;
+  background: #9f6c1c;
+  padding: 10px 10px;
+  border: none;
+  text-decoration: none;
+  justify-content: center;
+  margin: 0 auto;
+  font-size: 14px;
+  text-align: center;
+}
```
Comment (emarf) [inline style.css]: fix with common button

File: style.css [hunk 11]
```diff
@@ -217,3 +227,98 @@ body {
...
+.order-go__image-cat {
+  max-width: 184px;
+}
+
+/* Section - FOOTER */
+
+.footer {
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
```
Comment (emarf) [inline style.css]: right now you have display: block and this styles useless

File: style.css [hunk 12]
```diff
@@ -217,3 +227,98 @@ body {
...
+  background-color: #9f6c1c;
+  flex-direction: column;
```
Comment (emarf) [inline style.css:310]: same

File: style.css [hunk 13]
```diff
@@ -217,3 +227,98 @@ body {
...
+}
+
+.footer-title {
+  font-size: 14px;
+  text-transform: uppercase;
+  color: white;
+  margin-bottom: 15px;
+}
+
+.footer-button_link {
```
Comment (emarf) [inline style.css:330]: add transform: scale on hover effect

File: index.html (hunk 1 above)
Comment (senyka33) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (senyka33) [reply index.html]: fixed

File: style.css (hunk 13 above)
Comment (senyka33) [reply style.css:330]: add

**Pull request merged**